### PR TITLE
Adapt bridge to new API

### PR DESCRIPTION
**Pull request by raphaelrobert**

### CODE REVIEW

**Review by kkohbrok — COMMENTED** [review]
Looks good! Just one comment.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -6,21 +6,29 @@ name: Rust
...
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
+        with:
+          flutter-version: "3.x"
+          channel: "stable"
+          cache: true
+      - name: Build
+        run: cargo build --verbose
+      - name: Run tests
+        run: cargo test --verbose
```
Comment (kkohbrok) [inline .github/workflows/build.yml:30]: It might be good to have separate jobs for building and testing.

Comment (raphaelrobert) [reply .github/workflows/build.yml:30]: Sure, but then we would have to think about caching (if that's possible at all). The CI is pretty slow already, I think it's ok to keep it that way at this stage.

Comment (kkohbrok) [reply .github/workflows/build.yml:30]: Fair enough.

**Review by kkohbrok — APPROVED** [review]

**Pull request merged**